3p/3p.js

```diff
@@ -154,7 +154,7 @@ export function validateSrcPrefix(prefix, src) {
  * @param {string} src
  */
 export function validateSrcContains(string, src) {
-  if (src.indexOf(string) === -1) {
+  if (src.includes(string) === false) {
```

Comment:
Instead of `if (condition === false)`, please use `if (!condition)`. Likewise for the `true` case. It's more concise and easier to read!